usage_examples/streamlit/streamlit_app.py

```diff
@@ -0,0 +1,49 @@
+from spacy import util
+from spacy_streamlit import visualize_ner, visualize_textcat
+import streamlit as st
+import os
+
+DEFAULT_TEXT = "Ernest Hemingway, born in Illinois, is generally considered one of the best authors of his time."
+
+st.title("spacy-llm Streamlit Demo")
+st.markdown(
+    """
+    The [spacy-llm](https://github.com/explosion/spacy-llm) package integrates 
+    Large Language Models (LLMs) into spaCy, featuring a modular system 
+    for fast prototyping and prompting, and turning unstructured responses 
+    into robust outputs for various NLP tasks, no training data required.
+
+    This demo uses the OpenAI backend to demonstrate the NER and textcat 
+    tasks.
+    """
+)
+
+os.environ["OPENAI_API_KEY"] = st.text_input("Your OpenAI API key", type="password")
+text = st.text_area("Text to analyze", DEFAULT_TEXT, height=70)
+
+if os.environ.get("OPENAI_API_KEY"):
+    textcat_config = util.load_config(
+        "usage_examples/streamlit/openai_textcat_zeroshot.cfg"
+    )
+    textcat_model = util.load_model_from_config(textcat_config, auto_fill=True)
+
+    ner_config = util.load_config("usage_examples/streamlit/openai_ner_zeroshot.cfg")
+    ner_model = util.load_model_from_config(ner_config, auto_fill=True)
```

Comment:
Let's switch to `spacy_llm.util.assemble` here too (you'll still need the import further up).

```suggestion
    textcat_model = assemble(textcat_config)
    ner_model = assemble(ner_config)
```

Reply:
Then you'd have:

```suggestion
    textcat_config = util.load_config(TEXTCAT_CONFIG)
    textcat_model = assemble_from_config(textcat_config)

    ner_config = util.load_config(NER_CONFIG)
    ner_model = assemble_from_config(ner_config)
```

Reply:
I think `util.load_config` expects a `file` type input? Not a string?

Reply:
Ah, the `load_config_from_str` instead.